pkgs/retry/retry

```diff
@@ -4,6 +4,8 @@
 #
 set -euf
 
+export PATH=@coreutils@/bin"${PATH+:$PATH}"
```

Comment:
Cool Igor, but that prompts the next question: what sets `coreutils` here? Some `configure`?